packages/thirdweb/package.json

```diff
@@ -315,40 +279,40 @@
     "node": ">=18"
   },
   "devDependencies": {
-    "@aws-sdk/client-lambda": "3.659.0",
+    "@aws-sdk/client-lambda": "3.670.0",
     "@aws-sdk/credential-providers": "3.670.0",
-    "@biomejs/biome": "1.9.2",
+    "@biomejs/biome": "1.9.3",
     "@chromatic-com/storybook": "2.0.2",
     "@codspeed/vitest-plugin": "3.1.1",
     "@coinbase/wallet-mobile-sdk": "1.1.2",
     "@mobile-wallet-protocol/client": "0.0.3",
-    "@react-native-async-storage/async-storage": "1.24.0",
+    "@react-native-async-storage/async-storage": "2.0.0",
```

Comment:
need to double check this

Reply:
ah yeah lemme actually just downgrade this again